src/components/ebay-menu/test/mock/index.js

```diff
@@ -30,3 +30,19 @@ exports.Fake_2Items = assign({}, exports.Basic_2Items, {
         renderBody: createRenderBody(`Item text ${i}`)
     }))
 });
+
+function getCountry(num) {
```

Comment:
What's the point of having this as a function?
You can probably use an array instead with various countries. 
Like: 
const countries = ['Albania', 'Alcania', 'Something else'];

`item ${i} ${countries[i]}`;